zstacklib/zstacklib/utils/vm_metadata_handler.py

```
@@ -0,0 +1,68 @@
+class VmMetadataScanEntry(object):
+    __slots__ = (
+        'vmUuid', 'vmName', 'vmCategory', 'architecture',
+        'schemaVersion', 'metadataPath', 'sizeBytes', 'lastUpdateTime',
+        'incomplete',
+    )
```

Ruff (0.15.6)

[warning] `VmMetadataScanEntry.__slots__` is not sorted

Apply a natural sort to `VmMetadataScanEntry.__slots__`

(RUF023)